src/twilio_call_center/utils.py

```diff
@@ -2,6 +2,24 @@
 
 from django.conf import settings
 
+try:
+    from django_twilio.client import twilio_client
+    from django_twilio.decorators import twilio_view
```

Comment:
This whole hunk of changes is moved (almost identically to another file. See second commit for alterations to this code)